src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -157,28 +140,28 @@ public static void NonExistentAction_Throws_ActionNotSupportedException()
     [OuterLoop]
     public static void TimeoutTest_SendTimeout5Seconds()
     {
-        bool exceptionThrown = false;
+        BasicHttpBinding binding = new BasicHttpBinding();
+        binding.SendTimeout = TimeSpan.FromMilliseconds(5000);
+        ChannelFactory<IWcfService> factory = new ChannelFactory<IWcfService>(binding, new EndpointAddress(Endpoints.HttpBaseAddress_Basic));
+
+        Stopwatch watch = new Stopwatch();
         try
         {
-            BasicHttpBinding binding = new BasicHttpBinding();
-            binding.SendTimeout = TimeSpan.FromMilliseconds(5000);
-            ChannelFactory<IWcfService> factory = new ChannelFactory<IWcfService>(binding, new EndpointAddress(Endpoints.HttpBaseAddress_Basic));
-            IWcfService proxy = factory.CreateChannel();
-            proxy.EchoWithTimeout("Hello");
-        }
-        catch (TimeoutException)
-        {
-            exceptionThrown = true;
+            var exception = Assert.Throws<TimeoutException>(() =>
+            {
+                IWcfService proxy = factory.CreateChannel();
+                watch.Start();
+                proxy.EchoWithTimeout("Hello");
```

Comment:
Oh nevermind. I'm being dumb... yes. The service method should just wait a number of seconds. 

However, we run into an interesting case if the number passed then ends up exceeding the service's Send/ReceiveTimeout...